src/pages/Settings/UsersAndPermissions/Groups/index.tsx

```diff
@@ -126,6 +143,7 @@ const Groups = () => {
                   rows={2}
                   multiline
                 />
+                <Field name="permissions" component={RoleSelectField} predefinedRoles={activeRow?.permissions || []}/>
```

Comment:
I would rather use a `<FastField>` here because the rerender of the role select field is expensive and the other fields suffer. You can test it by typing in the name or the description.
Using `FastField`, it won't rerender when updating any of the other fields in the form.
https://formik.org/docs/api/fastfield
```suggestion
                <FastField name="permissions" component={RoleSelectField} predefinedRoles={activeRow?.permissions || []}/>
```

Reply:
Yeah, good catch, I will try to replace Field with FastField